### PR TITLE
add: Slider Component

### PR DESCRIPTION
**Pull request by shilpe26**
Added Slider component with features like:-
🔹Range from 0 to 600 value
🔹Each step of 50 value
slider

### CODE REVIEW

File: components/slider.html [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+                    sandbox="allow-scripts allow-same-origin">
+                    </iframe>
+                </div>
+                <!-- Footer -->
+                <footer>
+                    <div class="prev">
+                        <i class="fas fa-angle-left"></i>
+                        <a href="./grid.html">Grid</a></div>
+                    <!-- <div class="next"><a href="">Alert</a>
+                        <i class="fas fa-angle-right"></i>
```
Comment (anuj-kumary) [inline components/slider.html:120]: Remove unwanted comments

**Review by anuj-kumary — APPROVED** [review]
Everything looks good to me

File: components/slider.html (hunk 1 above)
Comment (shilpe26) [reply components/slider.html:120]: Will Do.✔️

File: components/slider.html [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+        <nav class="navigation">
+            <div class="nav-brand"><a class="link" href="/index.html">Rebalance-UI</a></div>
+            <ul class="non-bullet">
+                <li class="list-tem-inline"><a class="link link-active" href="/index.html">Home</a></li>
+                <li class="list-tem-inline"><a class="link" href="./avatar.html">Docs</a></li>
+            </ul>
+        </nav>
+    </header>
+
+    <!-- side-list bar -->
```
Comment (vinc3nati) [inline components/slider.html:35]: commented code makes it easier to scan, nice!

**Review by vinc3nati — APPROVED** [review]

File: components/slider.html (hunk 2 above)
Comment (shilpe26) [reply components/slider.html:35]: Thank you @vinc3nati

**Pull request merged**